internal/dto/semester_dto/semester.dto.go

```diff
@@ -58,3 +58,17 @@ type GetSemesterSwaggerOutput struct {
 	Success bool            `json:"message"`
 	Data    *SemesterOutput `json:"data"`
 }
+
+type SemesterOutputCount struct {
+    ID                   int64     `json:"id"`
+    Name                 string    `json:"name"`
+    StartTime            time.Time `json:"start_time"`
+    EndTime              time.Time `json:"end_time"`
+    CapstoneGroups       int64     `json:"capstone_groups"`       
```

Comment:
If this is describe the number of capstone groups need to name total capstone group